application/bulma-table-service/src/main/java/pl/ds/bulma/rest/table/AddTableRowRestAction.java

```diff
@@ -59,8 +59,7 @@ public RestActionResult<String> perform(AddTableRowRestModel addTableRowRestMode
       orderRow(parentNode, newRow, selectedRow, addTableRowRestModel.isInsertBefore());
 
       addTableRowRestModel.getResourceResolver().commit();
-      return RestActionResult.success("Table row created",
-          "New table row created at " + newRow.getPath(), newRow.getPath());
+      return RestActionResult.success("Table row created");
```

Comment:
a little update here because Michal mentioned we don't need the notification when table row is added, because it's obvious.